txt2tags.py

```diff
@@ -235,13 +235,13 @@
 """,
     "html": """\
 <!DOCTYPE html>
-<html>
+<html lang="en">
 <head>
 <meta charset="%(ENCODING)s">
 <title>%(HEADER1)s</title>
 <meta name="generator" content="https://txt2tags.org">
 <link rel="stylesheet" href="%(STYLE)s">
-<style type="text/css">
+<style>  
```

Comment:
Is it discouraged to add the CSS type explicitly?

Reply:
There is a warning:
-----------------------------------------
Warning: The type attribute for the style element is not needed and should be omitted.

From line 7, column 1; to line 7, column 23

ags.org">↩<style type="text/css">↩block